### ISSUE TITLE
Enable ruby19_syntax by default

### ISSUE THREAD

**Issue by dishcandanty** [amazing-print/amazing_print#47]
I imagine this is somewhat of a breaking change. But wondering if there's any reason why this wouldn't be enabled by default in 2020.

**Comment by fig** [comment]
Are you referring to symbols as Hash keys?

`{ :foo => 'bar' }` vs `{ foo: 'bar' }`

If so, the general fashion within the community appears to use 1.9 syntax for source code but use hash rockets for output. FWIW, I'm quite happy to keep it that way.

**Comment by dishcandanty** [comment]
Yup! `ruby19_syntax: false` => `ruby19_syntax: true`. I hadn't really paid attention if that was generally accepted within the community. I just like being able to copy and paste the output, without it being an immediate rubocop violation.

**Comment by HarlemSquirrel** [comment]
@paddor I'm OK making ruby19 syntax the default for the next minor version bump if you are but we may also want to rework this into something like `hash_format` to support JSON for #70

**Comment by gurgeous** [comment]
Let's do this! I forgot `ruby19_syntax` existed, and I just added it to my `aprc`. I also just discovered negative indent (to left justify), which is a much better default IMO. As long as I was messing with my aprc I did a few more things:

- Added custom RGB colors. This makes my terminal look 100x better.
- Hacked in some improvements to the default `inspect` formatter
- Removed `blank?` values from ActiveRecord models

What a difference, I should've made this change years ago. I've been using ap for well over a decade... See https://gist.github.com/gurgeous/b0045645a1c7f93194ca285ddebafad7

### new
<img width="443" alt="Image" src="https://github.com/user-attachments/assets/be8ec25d-c838-486d-b6c6-fc6207d8feeb" />

### old

<img width="495" alt="Image" src="https://github.com/user-attachments/assets/bfa91b3a-ea9d-45f6-baaa-1a9e3b14d056" />

**Comment by HarlemSquirrel** [comment]
Nice! I'm down with changing the default colors if it looks good in dark and light terminals

**Comment by gurgeous** [comment]
The user's ANSI colors are probably the only "safe" defaults, unfortunately. With some work it's possible to detect the terminal bg color and pick a nice theme (see https://github.com/gurgeous/table_tennis) but it's not for the faint of heart.

I think an easier change would be:

- Support for RGB colors. This is widely supported and real easy these days, see my aprc for a hacky example.
- Support for a `theme` config option. It defaults to `:ansi` but can be manually set to `:light` or `:dark`.

I also think ap should switch to `ruby19_syntax: true` and a `indent: -2` by default, but that's a separate issue... Let me know if you want help with anything. Thanks for all your work on amazing print!

**Comment by HarlemSquirrel** [comment]
That makes sense for the colors

**Comment by HarlemSquirrel** [comment]
For alignment, I'd rather have a separate option the left aligning hash keys so you could have a larger indent with left aligned hash keys

**Comment by HarlemSquirrel** [comment]
Oh never mind I forgot the negative number affects the indent size too lol

**Comment by gurgeous** [comment]
> Oh never mind I forgot the negative number affects the indent size too lol

I know, isn't that a wee bit confusing? I tried it on a lark and I found the output much easier to read for some reason. I should've tried it ages ago. That PR looks pretty good, love updating the default. Let's keep going!
